docs/schema.md

```diff
@@ -650,7 +650,10 @@ Only **a single type** (either `"introduced"`, `"fixed"`, `"last_affected"`,
 Entries in the `events` array can contain either `"last_affected"` or `"fixed"`
 events, but not both. It's **strongly recommended** to use `"fixed"` instead of
 `"last_affected"` where possible, as it precisely identifies the version which
-contains the fix.
+contains the fix. `"last_affected"` should be thought of as a hard ceiling. 
+Versions above `"last_affected"` are considered unaffected. Unfortunately this 
+opens up the possibility for false negatives, which is why `"fixed"` is 
+overwhelmingly preferred.  
```

Comment:
I think it's useful to emphasize that the use of `last_affected` is limited to specific situations where there is no known fix at the time of publication. I also would suggest slightly de-emphasizing that versions above are unaffected, as likely as that scenario is.

```suggestion
contains the fix. `"last_affected"` should be thought of as the hard ceiling 
of the vulnerability _at the time of publication_ in the absence of a fixed version.
Versions above `"last_affected"` should be considered unaffected. Unfortunately
this opens up the possibility for false negatives, which is why `"fixed"` is 
overwhelmingly preferred.
```

Reply:
Agreed. Thanks!